DDR4 OC Guide.md

```diff
@@ -222,66 +226,72 @@ As far as I know, tCL, tRCD, tRP and possibly tRFC can (or can not) see voltage
 ### Expected Max Frequency
 * Below are the expected max frequency for some of the common ICs:
 
-  | IC | Expected Max Frequency (MHz) |
-  | :-: | :-------------------------: |
-  | Hynix 8Gb AFR | 3600 |
-  | Hynix 8Gb CJR | 4000<sup>1</sup> |
-  | Hynix 8Gb DJR | 4000+ |
-  | Nanya 8Gb B-die | 4000+ |
-  | Micron 8Gb Rev. E | 4000+ |
-  | Samsung 4Gb E-die | 4000+ |
-  | Samsung 8Gb B-die | 4000+ |
-  | Samsung 8Gb D-die | 4000+ |
-  * <sup>1</sup>CJR is a bit inconsistent in my testing. I've tested 3 RipJaws V 3600 CL19 8GB sticks. One of them was stuck at 3600MHz, another at 3800MHz but the last could do 4000MHz, all at CL16 with 1.45v.
+  | IC | Effective Speed attainable at 1.35V (MT/s) | Expected Max Effective Speed (MT/s) |
+  | :-: | :-------------------------: | :------------: |
+  | Hynix 8Gb AFR | 3600 | 3600 |
+  | Hynix 8Gb CJR | 3800 | 4133<sup>1</sup> |
+  | Hynix 8Gb DJR | 4000+ | 5000+ |
+  | Nanya 8Gb B-die | 3600 | 4000+ |
+  | Micron 8Gb Rev. B | 3400 | 4000
```

Comment:
Source for Rev. B doing 4000?

Reply:
Micron Rev B or Nanya B? You put the Nanya B. As for Micron, they literally sell XMP kits that do more than 4000. Furthermore, Rev N is just a modded Rev B, so IC limitations are still the same more or less. Unless you mean I should put it at 5000+, in which case sure.

Reply:
> they literally sell XMP kits that do more than 4000

Of 8Gb Rev B?

> Rev N is just a modded Rev B

16Gb Rev. B not 8Gb.